lib/pull_preview/github_sync.rb

```diff
@@ -58,10 +58,13 @@ def self.clear_dangling_deployments(repo, app_path, opts)
         if pr.base.repo.owner.type == "Organization"
           fake_github_context.organization = pr.base.repo.owner
         end
+        environment = PullPreview.octokit.list_environments(repo, per_page: 100).environments.find { |env| env.name.includes("gh-pr-#{pr.number}") }
         if pr_issue.state == "closed"
           PullPreview.logger.warn "[clear_dangling_deployments] Found dangling #{label} label for PR##{pr.number}. Cleaning up..."
+          destroy_environment(repo, environment)
```

Comment:
this will already be called when setting the GitHub status to destroyed - https://github.com/pullpreview/action/blob/bcdf8162787c5d7efa6b11e14d86aa9b7ed9026c/lib/pull_preview/github_sync.rb#L271